src/utils/date.ts

```diff
@@ -17,18 +17,34 @@ import {
   parse,
   parseISO,
   set,
+  getSeconds,
 } from 'date-fns';
 import en from 'date-fns/locale/en-GB';
 import nb from 'date-fns/locale/nb';
 import humanizeDuration from 'humanize-duration';
-import {DEFAULT_LANGUAGE, Language} from '@atb/translations';
+
+import {
+  parse as parseIso8601Duration,
+  toSeconds as toSecondsIso8601Duration,
+} from 'iso8601-duration';
```

Comment:
It might not need to have another library, please check the library we are using now: https://date-fns.org/v2.30.0/docs/formatISO and test :)

Reply:
Not supported yet, but will be in V3 it seems:
https://github.com/date-fns/date-fns/issues/3150

Reply:
Please consider creating a task to change when version is released, fewer dependencies better :)

Reply:
https://github.com/AtB-AS/kundevendt/issues/7795